operator/controllers/seldondeployment_controller.go

```diff
@@ -1091,12 +1092,6 @@ func (r *SeldonDeploymentReconciler) createIstioServices(components *components,
 func (r *SeldonDeploymentReconciler) createServices(components *components, instance *machinelearningv1.SeldonDeployment, all bool, log logr.Logger) (bool, error) {
 	ready := true
 	for _, svc := range components.services {
-		if !all {
```

Comment:
@axsaucedo we are created the svc components without waiting for all pods to be ready as the explainer pod needs to send a dummy request via the svc to the graph